lib/developerTestAccounts.ts

```diff
@@ -15,8 +15,8 @@ export async function createDeveloperTestAccount(
   accountName: string
 ): Promise<DeveloperTestAccount> {
   try {
-    const resp = await _createDeveloperTestAccount(accountId, accountName);
-    return resp;
+    const { data } = await _createDeveloperTestAccount(accountId, accountName);
+    return data;
```

Comment:
I mentioned this to @joe-yeager, but these utils are such thin wrappers now that I think we should just delete the entire file. It's not really adding much value at all. We could probably delete lib/sandboxes too, since it's also not doing any additional logic in the api call wrappers.

We can do this in another PR though. It's possible Joe already did it in his error handling PR

Reply:
I haven't done it yet, but I'll add an issue so we don't lose track of it and do it in a follow up PR.